tasks/push-snapshot/README.md

```diff
@@ -11,6 +11,9 @@ Tekton task to push snapshot images to an image registry using `cosign copy`.
 | pushSourceContainer| Push the source container to the mapped target                            | Yes      | false                |
 | retries            | Retry copy N times                                                        | Yes      | 0                    |
 
+## Changes since 1.1.0
+* The source container source pullspec was updated to use a git sha tag instead of the image digest.
```

Comment:
What was the issue with using image digest? Was it in fact the digest of the binary container, so it wouldn't work for the source container?

Reply:
prior to this PR, we were trying to copy an image using the pullspec `image@sha256:fbe973077b00242eeb3d25101def1be286c621436c5a003527736eabb48f0398-src` which does not makes sense or exist.

what does exist is `image:<git-sha>-src`